simple-repository-API/StudentRepository.cs

```diff
@@ -6,35 +6,34 @@
 namespace simple_repository_API
 
 {
-    public class Database_Connection
+    public class StudentRepository
     {
-        public List<Student> Database_connect() 
+        public Student GetStudent(int id) 
         {
-            List <Student> students = new List <Student>();
-
+            Student student = new Student();
             using (SqlConnection connection = new SqlConnection(
                    @"Server = Laptop-SS4D3ECJ\SQLEXPRESS; Database = school; Trusted_Connection = True;"))
             {
                 connection.Open();
-                string queryString = "SELECT ID_Student,Name,Surname,Age FROM Student";
-                SqlCommand command = new SqlCommand(queryString, connection);
+                string query = "SELECT Id_Student,Name,Surname,Age FROM Student WHERE Id_Student = @Id_Student";
```

Comment:
To jeszcze nie ma prawa zadziałać. Trzeba dodać logikę dotyczącą przechwytywania poprawnego parametru do querki